lib/src/key/mod.rs

```diff
@@ -4,26 +4,28 @@
 /// crate::key::root::hb                 /!hb{ts}/{nd}
 /// crate::key::root::nd                 /!nd{nd}
 /// crate::key::root::ni                 /!ni
-/// crate::key::root::ns                 /!ns{ns}
+/// crate::key::root::ns                 /!ns{ns name}
```

Comment:
This is for storing namespace aliases.
If we are to implement `RENAME NAMESPACE foo TO bar` or something like that,
it should work by first deleting`/!ns{foo}` and then putting `/!ns{bar}` in a transaction.
It's one-del one-put transaction, making the renaming cheap.